libraries/networking/src/udt/SendQueue.cpp

```diff
@@ -24,7 +25,27 @@
 #include "Socket.h"
 
 using namespace udt;
-using namespace std::chrono;
+
+// This class is not thread-safe
```

Comment:
@birarda Going over the class I realized it is not thread safe, is it a requirement for `std::lock()`?